ci-operator/templates/openshift/openshift-azure/build-azure-image.yaml

```diff
@@ -22,7 +22,7 @@ parameters:
 - name: IMAGE_ANSIBLE
   required: true
 - name: DEPLOY_OS
-  value: "centos7"
+  value: "rhel7"
```

Comment:
is this something we will pass into the template?

Reply:
For now it's hardcoded but in order to reuse the same template, I am thinking we can mount a secret that holds the OS and version that we want to build. That secret gets passed in this pod via ci-operator's --secret-dir flag so each job spec will use its own secret.